.coderabbit.yaml

```
@@ -0,0 +1,17 @@
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
+  review_status: true
+  collapse_walkthrough: false
+  auto_review:
+    enabled: true
+    drafts: false
+chat:
+  auto_reply: true
+exclude_paths:
+  - "prisma/migrations/**"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)